source/Calamari.Tests/KubernetesFixtures/KubernetesContextScriptWrapperLiveFixtureBase.cs

```diff
@@ -176,10 +176,17 @@ protected void DoDiscovery(AwsAuthenticationDetails authenticationDetails)
 
                 variables.Save(variablesFile.FilePath);
 
-                return InvokeInProcess(Calamari()
-                                       .Action(KubernetesDiscoveryCommand.Name)
-                                       .Argument("variables", variablesFile.FilePath)
-                                       .Argument("extensions", string.Join(',', extensions)));
+                var result = InvokeInProcess(Calamari()
+                       .Action(KubernetesDiscoveryCommand.Name)
+                       .Argument("variables", variablesFile.FilePath)
+                       .Argument("extensions", string.Join(',', extensions)));
+
+                foreach (var message in Log.Messages)
+                {
+                    Console.WriteLine($"[{message.Level}] {message.FormattedMessage}");
+                }
```

Comment:
Given that this uses the base class's Log property (and that this is a dupe of code above), is it worth creating a method to do this in that base class (ie in CalamariFixture)?  I assume this functionality would come in handy in other tests.

Reply:
I see what you're saying but in `CalamariFixture` the logger is an `InMemoryLogger` which actually does log to StdOut and StdErr as well. In this base class, the Logger is changed to a `DoNotDoubleLog` logger which overrides the writing to StdOut and StdErr so that the logging can be done manually. Basically `CalamariFixture` by default doesn't have this issue, only `KubernetesContextScriptWrapperLiveFixtureBase`. I can, however move the dupe code to a method in this class which I will do for neatness 👍

Reply:
Oh sorry, I missed that this used `KubernetesContextScriptWrapperLiveFixtureBase` - my bad 🙇

Reply:
No problems! the naming of the loggers and general structure of these tests doesn't make it super easy to understand 😅